data-raw/ctis_covid_behaviours.R

```diff
@@ -20,7 +24,7 @@ behav_ind_distancing <- covidcast(
   fetch() %>%
   select(geo_value, time_value, distancing = value)
 
-pop_dat <- state_census %>% select(abbr, pop)
+pop_dat <- state_census %>% select(abbr = ABBR, pop = POPESTIMATE2019)
```

Comment:
`pop_dat` isn't used, just delete it?  (And if it were used, there's probably some missing `tolower` on `ABBR`.)

After deletion, is `library(covidcast)` still needed?